src/components/form/range/dual_range.tsx

```diff
@@ -476,7 +484,7 @@ export class EuiDualRangeClass extends Component<
         min={min}
         max={Number(this.upperValue)}
         step={step}
-        value={this.lowerValue}
+        value={this.lowerValue === min ? '' : this.lowerValue}
```

Comment:
I don't think this change is right - if a user is deliberately selecting the minimum value, we shouldn't be showing an empty input.

Combined with the change above, if we want to support initial empty inputs, a better way of handling this is likely:

```suggestion
        value={this.props.value?.[0]}
```

Reply:
Oops, sorry, I missed the subsequent change on this line to check for a placeholder. I still don't think that UX is right or makes sense however, and my change request remains the same. A minimum value is a perfectly valid value to select. I would expect the minimum value to be displayed in the input if the user selects it in the range.

Reply:
I agree, it's not a friendly UX bahavior. I like your idea with `value={this.props.value?.[0]}` and letting people to pass an empty string if they want to replicate the effect from the issue!